docs/maps/mapPoint-series.md

```diff
@@ -1,7 +1,13 @@
 Map point series
-==========
+================
 
-A mappoint series is a special form of scatter series 
-where the points can be laid out in map coordinates on top of a map.
+A `mappoint` series is a special form of `scatter` series where the points can be laid out in map coordinates on top of a map.
 
-<iframe style="width: 100%; height: 520px; border: none;" src="https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/maps/demo/mappoint-latlon" allow="fullscreen"></iframe>
\ No newline at end of file
+<iframe style="width: 100%; height: 520px; border: none;" src="https://highcharts.com/demo/maps/mappoint-latlon" allow="fullscreen"></iframe>
```

Comment:
The SRC should use the URL for the embedded demo (`https://www.highcharts.com/samples/embed/maps/demo/mappoint-latlon`)